test/Integration/modules/edgecore/eventbus/eventbus_test.go

```diff
@@ -0,0 +1,214 @@
+/*
+Copyright 2019 The KubeEdge Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package e2e
+
+import (
+	"database/sql"
+	"encoding/json"
+	MQTT "github.com/eclipse/paho.mqtt.golang"
+	_ "github.com/mattn/go-sqlite3"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"os"
+	"path/filepath"
+	"time"
+
+	"github.com/kubeedge/kubeedge/test/Integration/utils/common"
+)
+
+//DeviceUpdate device update
+type DeviceUpdate struct {
+	State string `json:"state,omitempty"`
+}
+
+//Function to access the edge_core DB and return the device state.
+func checkDeviceState(deviceId string) string {
+
+	var id []uint8
+	var name string
+	var Description string
+	var State string
+	var LastOnline []uint8
+
+	pwd, err := os.Getwd()
+	if err != nil {
+		common.InfoV6("Failed to get PWD: %v", err)
+		os.Exit(1)
+	}
+	destpath := filepath.Join(pwd, "../../edge.db")
+	db, err := sql.Open("sqlite3", destpath)
+	if err != nil {
+		common.InfoV6("Open Sqlite DB failed :%v", err)
+	}
+	defer db.Close()
+	state, _ := db.Query("SELECT * FROM device", 0)
+	if err != nil {
```

Comment:
this err is from the previous open statement. why did we omit the error from the select ?

Reply:
this code queries devices from db right ? why have we received the results in a variable called "state" ?